### PR TITLE
Fix typo in api documentation about media for the PHP client

### PR DESCRIPTION
**Pull request by ahocquard**

### CODE REVIEW

File: content/php-client/ce-resources/media-files.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ You can get more information about this method [here](/php-client/list-resources
 #### With a cursor
 
 This method allows to iterate the media files. It will automatically get the next pages for you.
-With this method, it's not possible to get the previous page, or getting the total number of products.
+With this method, it's not possible to get the previous page, or getting the total number of medias.
```
Comment (tamarasaurus) [inline content/php-client/ce-resources/media-files.md]: media is already plural, you could say media files instead

Comment (NolwennP) [reply content/php-client/ce-resources/media-files.md]: I prefere `Media files`

Comment (ahocquard) [reply content/php-client/ce-resources/media-files.md]: you can gtm and trust me to fix it @NolwennP 😺

Comment (NolwennP) [reply content/php-client/ce-resources/media-files.md]: Never! 😈

**Pull request merged**